packages/blockchain-extension/extension/commands/importSmartContractPackageCommand.ts

```diff
@@ -75,7 +75,18 @@ export async function importSmartContractPackageCommand(): Promise<void> {
         });
 
         if (existingPackages.length > 0) {
-            throw new Error(`Package with name ${newName} already exists`);
+
+            const warnOverwrite: vscode.MessageItem = await vscode.window.showWarningMessage(`${newName} already exists. Would you like to replace the existing package? (You can set 'ibm-blockchain-platform.ext.showWarningOnPackageOverwrite: false' to make this the defualt).`, { title: UserInputUtil.OPEN_SETTINGS }, { title: UserInputUtil.CANCEL }, { title: UserInputUtil.REPLACE });
```

Comment:
`defualt` should be `default`

Reply:
I also think we should be doing something similar to 'package', where we check if the user has `ibm-blockchain-platform.ext.showWarningOnPackageOverwrite: false` and just overwrite it.